src/ui/popup.js

```diff
@@ -443,7 +443,9 @@ export default class Popup extends Evented {
     addClassName(className: string) {
         if (this._container) {
             this._container.classList.add(className);
-        }
+        } else if (this.options.className && this.options.className !== ``) {
```

Comment:
Nit: I believe `''` is falsey so that it's impossible to pass the first condition but not the second.

```suggestion
        } else if (this.options.className) {
```